public/gameInfo.php

```diff
@@ -676,9 +676,16 @@ function ResetProgress() {
             $imageIngame = media_asset($gameData['ImageIngame']);
             $pageTitleAttr = attributeEscape($pageTitle);
 
+            $fallBackConsoleIcon = "assets/images/system/unknown.png";
```

Comment:
This isn't being used. Try visiting any game page for console 52.

Reply:
And it needs to use the `asset()` helper too. As does the fallbackConsoleIcon defined in `gameList.php` (if you don't mind).

Reply:
I forgot the `onerror` event 🤦‍♂️
All of this feedback should be addressed in https://github.com/RetroAchievements/RAWeb/pull/1458/commits/b4c866f1317d94bc26c83926e1a130428148b3cc.